app/views/settings/_show_mobile.html.erb

```diff
@@ -1,14 +1,21 @@
 <%= render "shared/components/mobile_layout" do %>
   <h1 class="text-sm font-medium">User settings</h1>
+  <div class="mt-4 bg-line-colour-light rounded">
+    <% if "banner"=="banner" %>
```

Comment:
remove the if else condition and user team_banner helper method here. You need to add the placeholder file name in constants.rb STATIC_ASSETS